x/ecocredit/server/core/bridge.go

```diff
@@ -20,12 +20,15 @@ func (k Keeper) Bridge(ctx context.Context, req *core.MsgBridge) (*core.MsgBridg
 	}
 
 	sdkCtx := types.UnwrapSDKContext(ctx)
-	if err = sdkCtx.EventManager().EmitTypedEvent(&core.EventBridge{
-		Target:    req.Target,
-		Recipient: req.Recipient,
-		Contract:  req.Contract,
-	}); err != nil {
-		return nil, err
+	for _, credit := range req.Credits {
+		if err = sdkCtx.EventManager().EmitTypedEvent(&core.EventBridge{
+			Target:    req.Target,
+			Recipient: req.Recipient,
+			Contract:  req.Contract,
+			Amount:    credit.Amount,
```

Comment:
im wondering if maybe we should put batch denom in here? thinking in terms of a service that's subscribed to this event, would likely want access to the denom to extract the underlying credit info

Reply:
We only bridge back to a contract that credits have been bridged from and we don't populate or update any information on the receiving end so as far as I understand we don't need to access any credit info.

Reply:
The main goal of storing the contract on the ledger was to prevent the need for additional queries. If there is credit info that we need, we should be including the info in the event, but I don't believe there is.

Reply:
ahhh right, forgot it was only back to src 😸